rfcs/0007-service-date-filter.md

```diff
@@ -0,0 +1,102 @@
+# RFC Proposal
+[RFC Proposal]: #rfc-proposal
+
+* RFC Proposal ID: `0007-service-date-filter` (fill me in with a unique ident)
+* Start Date: 2020-10-15
+* RFC PR: [#376](https://github.com/CMSgov/beneficiary-fhir-data/pull/376)
+* JIRA Ticket(s):
+    * [BCDA-3871](https://jira.cms.gov/browse/BCDA-3871)
+
+This proposal will suggest adding an additional date range query on service completion date to EOB requests. This will allow users to guarantee that claims data falls within an expected date range.
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+- [RFC Proposal](#rfc-proposal)
+  - [Table of Contents](#table-of-contents)
+  - [Motivation](#motivation)
+  - [Proposed Solution](#proposed-solution)
+    - [Proposed Solution: Detailed Design](#proposed-solution-detailed-design)
+    - [Proposed Solution: Unresolved Questions](#proposed-solution-unresolved-questions)
+    - [Proposed Solution: Drawbacks](#proposed-solution-drawbacks)
+    - [Proposed Solution: Notable Alternatives](#proposed-solution-notable-alternatives)
+  - [Prior Art](#prior-art)
+  - [Future Possibilities](#future-possibilities)
+  - [Addendums](#addendums)
+
+## Motivation
+[Motivation]: #motivation
+
+BCDA needs to ensure that they are not returning beneficiary data to an ACO to which that beneficiary is not attributed. There are a variety of use-cases where this could occur if not handled properly:
```

Comment:
Should we list ACO Termination here too?

Reply:
I wast thinking that would be a "Future" use case since there may be some nuances with that.